onnx/test/basic_test.py

```diff
@@ -2,12 +2,41 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+
 from onnx.onnx_pb2 import AttributeProto, NodeProto, GraphProto, ModelProto, IR_VERSION
+
+import io
+import onnx
+import tempfile
 import unittest
 
 
 class TestProtobufExists(unittest.TestCase):
 
+    def test_load(self):
+        # Create a model proto.
+        model = ModelProto()
+        model.ir_version = IR_VERSION
+        model_string = model.SerializeToString()
+
+        # Test if input is string
+        loaded_model = onnx.load_from_string(model_string)
+        self.assertTrue(model == loaded_model)
+
+        # Test if input has a read function
+        f = io.BytesIO(model_string)
+        f.flush()
+        loaded_model = onnx.load(f)
+        self.assertTrue(model == loaded_model)
+
+        # Test if input is a file name
+        f = tempfile.NamedTemporaryFile()
+        f.write(model_string)
+        f.flush()
+        loaded_model = onnx.load(f.name)
```

Comment:
this will cause the re-open issue on windows, because inside `onnx.load` it will try to open the file again.

Reply:
Thanks! Yes, I saw it. Let me think of a solution. :-)

Reply:
You can create `tempfile.NamedTemporaryFile(delete=False)` and then close it and pass its name to `onnx.load`